packages/manager/apps/dedicated/client/app/billing/main/history/billing-main-history.routes.js

```diff
@@ -1,26 +1,157 @@
-import controller from './billing-main-history.controller';
-import template from './billing-main-history.html';
-
-angular.module('Billing').config(($stateProvider) => {
-  $stateProvider.state('app.account.billing.main.history', {
-    url: '/history?filters',
-    params: {
-      filters: {
-        value: '[]',
-        squash: true,
+import { Bill, DebtAccount } from '@ovh-ux/manager-models';
+import { ListLayoutHelper } from '@ovh-ux/manager-ng-layout-helpers';
+import get from 'lodash/get';
+import omit from 'lodash/omit';
+import partition from 'lodash/partition';
+
+const mapDateFilter = (comparator, value) => {
+  switch (comparator) {
+    case 'isAfter':
+      return { 'date.from': value };
+    case 'isBefore':
+      return { 'date.to': value };
+    case 'is':
+      return {
+        'date.from': value,
+        'date.to': moment(value)
+          .add(1, 'day')
+          .format('YYYY-MM-DD'),
+      };
+    default:
+      return {};
+  }
+};
```

Comment:
What about having this filter located in https://github.com/ovh/manager/blob/master/packages/manager/modules/filters ?
I guess it should be re-used elsewhere. 

(Not blocking for you PR)

Reply:
@JeremyDec I didn't even know that package existed 😅 
I partially agree because this is also related to the used API so to be reused, APIs have to behave the same for all date filters